entrypoint.sh

```diff
@@ -235,7 +240,7 @@ exec /usr/bin/subspace \
     "--http-host=${SUBSPACE_HTTP_HOST}" \
     "--http-addr=${SUBSPACE_HTTP_ADDR}" \
     "--http-insecure=${SUBSPACE_HTTP_INSECURE}" \
-    "--backlink=${SUBSPACE_BACKLINK}" \
+    "--subdir=${SUBSPACE_URL_SUBDIRECTORY}" \
```

Comment:
this is possibly a breaking change. Maybe remap internally backling to subdir, for backwards compatibility and easy upgrade? A deprecation notice could also be implemented for `--backlink`